docs/topics/secured-mission-common-interaction.adoc

```diff
@@ -2,33 +2,103 @@
 
 The Secured booster provides a default HTTP endpoint that accepts `GET` requests if the caller is authenticated and authorized. The `sso` directory of the booster contains a Java client you can use to interact with the Secured booster. The client first authenticates against the {RHSSO} server and then performs a `GET` request against the Secured booster using the access token returned by the authentication step.
 
-The Java client is packaged as an executable JAR. It is located at `target/sso-client.jar` and takes arguments from the CLI. You can use the `--help` argument to see a full list of available arguments.
+[[secured-endpoint]]
+== The Secured Endpoint
+When using a client to interact with the booster, you must specify the Secured booster endpoint, which is the Greeting servic). You can get a list of the endpoints using the `oc get routes` command. The following table shows sample output from the `oc get routes` where you can obtain the Secured booster endpoint name (`<SERVICE_HOST>`).
```

Comment:
Probably reword to mention that this way, you figure out _both_ the URL of the SSO server and of the secured application.